assets/js/components/utils/index.js

```diff
@@ -24,23 +20,21 @@ const getProductsRequests = ( {
 	queryArgs = [],
 } ) => {
 	const defaultArgs = {
-		per_page: IS_LARGE_CATALOG ? 100 : -1,
+		per_page: IS_LARGE_CATALOG ? 100 : 0,
```

Comment:
Wouldn’t 0 return no product?

Reply:
Nope. I noticed some functions treat 0 as "unlimited" and adopted that instead of -1 which also makes validation easier.